lua/jj/cmd.lua

```diff
@@ -170,6 +170,27 @@ local function handle_log_enter()
 		close_terminal_buffer()
 	end
 end
+
+--- Handle keypress n on `jj log` buffer to create a new commit after the current one
+local function handle_log_new()
+	local line = vim.api.nvim_get_current_line()
+
+	local revset = get_rev_from_log_line(line)
+
+	if revset then
+		-- If we found a revision, edit it
+		local cmd = string.format("jj new -A %s", revset)
+		local _, success = utils.execute_command(cmd, string.format("Error creating new change after `%s`", revset))
+		if not success then
+			return
+		end
+
+		utils.notify(string.format("Succesfully created change after: `%s`", revset), vim.log.levels.INFO)
+		-- Update the log buffer
+		M.log()
+	end
+end
+
 ---
```

Comment:
should we also add a new-before

```suggestion
--- Handle keypress n on `jj log` buffer to create a new commit before the current one
local function handle_log_new_before()
	local line = vim.api.nvim_get_current_line()

	local revset = get_rev_from_log_line(line)

	if revset then
		-- If we found a revision, edit it
		local cmd = string.format("jj new -B %s", revset)
		local _, success = utils.execute_command(cmd, string.format("Error creating new change after `%s`", revset))
		if not success then
			return
		end

		utils.notify(string.format("Succesfully created change before: `%s`", revset), vim.log.levels.INFO)
		-- Update the log buffer
		M.log()
	end
end
```